components/blitz/src/ome/formats/importer/cli/CommandLineImporter.java

```diff
@@ -492,19 +488,31 @@ public static void advUsage() {
     private static List<Annotation> toTextAnnotations(
                    List<String> namespaces, List<String> strings)
     {
-        if (namespaces.size() != strings.size())
-        {
-            throw new IllegalArgumentException(String.format(
-                            "#Namespaces:%d != #Text:%d", namespaces.size(),
-                            strings.size()));
-        }
         List<Annotation> annotations = new ArrayList<Annotation>();
-        for(int i = 0; i < namespaces.size(); i++)
+        CommentAnnotationI annotation;
+        if (namespaces.size() != strings.size())
```

Comment:
used this build : https://ci.openmicroscopy.org/job/OMERO-DEV-merge-build/lastSuccessfulBuild/ICE=3.6,jdk=8_LATEST,label=octopus/

And seems to be throwing out an error from an outdated version of the code, am i using the wrong build?